source/test_env.cpp

```diff
@@ -20,27 +20,30 @@
 #include <string.h>
 #include "greentea-client/test_env.h"
 
+// Auxilary macros
+#define NL "\n"
+
 // Generic test suite transport protocol keys
-const char* TEST_ENV_END = "end";
-const char* TEST_ENV_EXIT = "__exit";
-const char* TEST_ENV_SYNC = "__sync";
-const char* TEST_ENV_TIMEOUT = "__timeout";
-const char* TEST_ENV_HOST_TEST_NAME = "__host_test_name";
+const char* GREENTEA_TEST_ENV_END = "end";
```

Comment:
why is `` end `` and not ``__end`` ? trying to understand diff betwen this, success and failure and then the rest with ``__rest_of_keys``